.circleci/config.yml

```diff
@@ -341,6 +341,35 @@ jobs:
           - SERVICES=
           - PLUGINS=winston
 
+  typescript:
```

Comment:
When does this get run by CircleCI?

Reply:
Can we persist the results of building the docs to the workspace so we can view the generated docs in CircleCI before merging?

Reply:
Very good point, I will add that.